### PR TITLE
db/navigator-climate-demo

### PR DESCRIPTION
**Pull request by karolkolodziej**

### CODE REVIEW

**Review by sebastianbochan — APPROVED** [review]

**Comment by bre1470** [comment]
Looks good so far. Will you do the interactivity points after my PR of the detail chart, or should I wait for this one to merge?

**Comment by karolkolodziej** [comment]
I can make it after yours PR :) feel free to merge yours :)

**Comment by bre1470** [comment]
Okay. Map PR is merged now. :)

File: samples/dashboard/demos/climate/demo.css
Comment (sebastianbochan) [inline samples/dashboard/demos/climate/demo.css:27]: I would prefer to use hex everywhere instead of mixing static values and hex.

**Review by sebastianbochan — CHANGES_REQUESTED** [review]

**Review by sebastianbochan — APPROVED** [review]

**Review by bre1470 — APPROVED** [review]
There are some visual bugs in some ranges, but this seems to be something in Highcharts.

**Pull request merged**
